### DISCUSSION TITLE
Definitive branding

### DISCUSSION THREAD

**Discussion by jurrejansen** [Shelf-nu/shelf.nu#32] [Ideas]
Hi @carlosvirreira @DonKoko 

As suggested by @DonKoko I'm moving this discussion to a separate issue. Would be nice to update the production ready designs to the definitive branding.

ORIGINALS

![logo](https://user-images.githubusercontent.com/76112272/226338259-d20095a2-5760-4a1a-8706-813484ed873c.jpg)
![logo-1](https://user-images.githubusercontent.com/76112272/226338248-7067fc8c-07f2-4548-a29c-81f0b08b720a.jpg)
![logo-2](https://user-images.githubusercontent.com/76112272/226338254-a0dce358-a0d7-4a27-bb65-50cb223f49c9.jpg)
![logo-3](https://user-images.githubusercontent.com/76112272/226338257-13f7a169-e020-47bc-a051-bcd3270fc83d.jpg)

Based on our discussion last week I think we compromised on a few things.
- We use all lowercase type for the logo
- We stick to the original (S-shaped) icon

UPDATED ORIGINALS

![logo-1](https://user-images.githubusercontent.com/76112272/226337949-5be95c4c-fab9-401f-8b9f-d6799fe8704a.jpg)
![logo-2](https://user-images.githubusercontent.com/76112272/226337952-a5bd7f9b-8e11-4e2b-a75e-e60daabc76a2.jpg)
![logo-3](https://user-images.githubusercontent.com/76112272/226337956-e5cc4aa4-f89a-4f0b-938c-cc8cb8dc85eb.jpg)
![logo](https://user-images.githubusercontent.com/76112272/226337958-3b2788ee-02c0-4c28-8334-58985bcb425d.jpg)

To be discussed:
- Color composition of logo
- Color composition of dashboard UI

**Comment by carlosvirreira** [comment]
Reflecting on competitive landscape I am willing to take the lower case S version. The rest for me is all good to go. 

Why? 
- It feels a bit friendlier
- it does not lose its professional look
- it feels a bit less rigid than the competition 

 Curious on @DonKoko thinking here.

**Comment by jurrejansen** [comment]
I'm personally not a fan of orange, but i think both of you do like it. However I think it does not go well at all with the dark blue. My suggestion down below. Slightly changed the orange + using a shade of black insead of blue.

![logo-1](https://user-images.githubusercontent.com/76112272/226342814-72a0338c-263d-4665-9a65-84ce5eb99ed6.jpg)
![logo-2](https://user-images.githubusercontent.com/76112272/226342819-fbb19eac-5d57-40ec-944d-f1c45722cdf8.jpg)
![logo-3](https://user-images.githubusercontent.com/76112272/226342821-b2756f10-5bf3-4045-b693-1a1621154474.jpg)
![logo](https://user-images.githubusercontent.com/76112272/226342823-7f204901-a6b8-491c-b9ee-77f491272034.jpg)

As for color composition for dashboard I would suggest these. i like the light themed one more.

![Desktop](https://user-images.githubusercontent.com/76112272/226342987-c0512b91-9757-47a4-876d-cd37fa49e79d.jpg)

![Desktop-1](https://user-images.githubusercontent.com/76112272/226342968-8ce1209c-64c5-495a-8cd9-adc1d629a5ae.jpg)

![Desktop](https://user-images.githubusercontent.com/76112272/226343050-21740843-3de8-479b-beb7-1bd23391a74b.jpg)

![Desktop-1](https://user-images.githubusercontent.com/76112272/226343036-97243a30-40f8-4aec-8e94-0e6b754b166e.jpg)


Here's the page in which i was messing around with different options if you wanna check. You can also check the version above there. It's a bit nicer to see there than these exports i shared here.

https://www.figma.com/file/2XkiKJ4II7Mux6JJL9hYTm/Shelf-App-%7C-Production-Ready?node-id=5452-327512

**Reply by DonKoko** [reply]
I also like this more. The orange i see is different in the layouts and in the logo tho but I definitely like the lower case way more and I agree it works better with the black. 

As for the page layout, I also like it and I think we have to take a decision by now and move forward. I also like the light one but honestly its too much of a flashbang for me. I never use anything in light mode and this makes it a bit difficult.

Also the sidebar is bothering me somehow in the white version. I am not 100% sure why or what but it feel like I cant see whats going on there. Maybe we can try to use at least some shade on the BG of the sidebar, even if its almost white. Like `gray-25` or even `primary-25` or `primary-10`

Here is for example how they are doing it in GH:
![image](https://user-images.githubusercontent.com/7840007/226555994-164299f2-c3e3-4c2b-bdc0-99badfd90618.png)

In dark mode they are also 2 different shades.

**Comment by carlosvirreira** [comment]
> I'm personally not a fan of orange, but i think both of you do like it. However I think it does not go well at all with the dark blue. My suggestion down below. Slightly changed the orange + using a shade of black insead of blue.
> 
> ![logo-1](https://user-images.githubusercontent.com/76112272/226342814-72a0338c-263d-4665-9a65-84ce5eb99ed6.jpg) ![logo-2](https://user-images.githubusercontent.com/76112272/226342819-fbb19eac-5d57-40ec-944d-f1c45722cdf8.jpg) ![logo-3](https://user-images.githubusercontent.com/76112272/226342821-b2756f10-5bf3-4045-b693-1a1621154474.jpg) ![logo](https://user-images.githubusercontent.com/76112272/226342823-7f204901-a6b8-491c-b9ee-77f491272034.jpg)
> 
> As for color composition for dashboard I would suggest these. i like the light themed one more.
> 
> ![Desktop](https://user-images.githubusercontent.com/76112272/226342987-c0512b91-9757-47a4-876d-cd37fa49e79d.jpg)
> 
> ![Desktop-1](https://user-images.githubusercontent.com/76112272/226342968-8ce1209c-64c5-495a-8cd9-adc1d629a5ae.jpg)
> 
> ![Desktop](https://user-images.githubusercontent.com/76112272/226343050-21740843-3de8-479b-beb7-1bd23391a74b.jpg)
> 
> ![Desktop-1](https://user-images.githubusercontent.com/76112272/226343036-97243a30-40f8-4aec-8e94-0e6b754b166e.jpg)
> 
> Here's the page in which i was messing around with different options if you wanna check. You can also check the version above there. It's a bit nicer to see there than these exports i shared here.
> 
> https://www.figma.com/file/2XkiKJ4II7Mux6JJL9hYTm/Shelf-App-%7C-Production-Ready?node-id=5452-327512

This one hits the right strings for me. I would approve this one and go with it until v2.

**Comment by DonKoko** [comment]
I have transformed the issue to an actual organization wide discussion.

**Comment by carlosvirreira** [comment]
Glad we took a decision today team. 

@jurrejansen please update the latest branding and UI on https://github.com/Shelf-nu/Branding
